src/backend/vulkan/src/window.rs

```diff
@@ -164,15 +165,13 @@ impl Instance {
             unsafe { w_loader.create_wayland_surface(&info, None) }.expect("WaylandSurface failed")
         };
 
-        self.create_surface_from_vk_surface_khr(surface, width, height, 1)
+        self.create_surface_from_vk_surface_khr(surface)
     }
 
     #[cfg(target_os = "android")]
     pub fn create_surface_android(
```

Comment:
The calls to `create_surface_from_wayland` and `create_surface_android` need to be updated to remove the width/height parameters.